eng/SharedFramework.External.props

```diff
@@ -6,6 +6,19 @@
 -->
 <Project>
 
+  <!-- For the targeting pack, always use packages with PatchVersion=0 -->
+  <PropertyGroup Condition="'$(MSBuildProjectName)' == 'Microsoft.AspNetCore.App.Ref'">
```

Comment:
We might need to change this to downgrade these versions if `IsReferenceAssemblyProject=true` (or if the project is the ref pack), otherwise the ref pack will wind up transitively referencing the higher versions of these guys. @dougbu @JunTaoLuo thoughts?